lib/tasks/update-config-xml-android-version-code.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var fs      = require('fs');
+var path = require('path');
+var Promise = require('../ext/promise');
```

Comment:
'Promise' is defined in a future version of JavaScript. Use a different variable name to avoid migration issues.<br>'require' is not defined.